examples/integrations/neo4j/neo4j-mp/src/test/java/io/helidon/examples/integrations/neo4j/mp/MainTest.java

```diff
@@ -38,7 +38,7 @@
  * Main tests of the application done here.
  */
 @Disabled("3.0.0-JAKARTA") // OpenAPI
-// Caused by: java.lang.NoSuchMethodError: 'java.util.List org.jboss.jandex.ClassInfo.unsortedFields()'
+// Caused by: java.lang.NoSuchMethodError: 'java.util.List io.smallrye.jandex.ClassInfo.unsortedFields()'
```

Comment:
(Just double-checking that you meant `io.smallrye` here and not a historical reference to `org.jboss`.)

Reply:
Right you are. Global search and replace gone rogue. (Although they changed the Maven coords they left the package names alone.)